### ISSUE TITLE
calculate the loss function for each neuron in the output layer

### ISSUE THREAD

**Issue by Manal4** [keras-team/keras#13919]
Hi

I have a regression problem, and I used a Sequential model where the shape of the input layer is (batch, sequence, n)  and the shape of the output layer is (batch, sequence, m),  and there are another two hidden layers. My question is that Keras calculates the loss( e.g.,  MSE : the mean squared error) between y-pred and y-true. IS there a way to calculate the loss over each target (in this example m targets 'neurons').

Is there any reference about how the loss is computed over the neurons of the output layer? 
I understand that if there are multiple output layers, the loss function will be the mean of losses  but if I have only one output layer how can I compute the loss for each target?  I tried to compute it using the MSE between the predictions and the true values for each target but I could not see how these values are related to the MSE from the model evaluation on the test data.

For example:

             ` loss MSE (test-set): [0.011158590205013752]`

where the losses, MSEs from predictions:

             ` target 0:  0.1327250661730257
              target 1 0.44203967357324886
              target 2: 0.1728217512614996
              target: 3: 2.2613116011898033
              target 4: 1.125334612706769
              target 5: 1.0468472140222114`  

I would appreciate if there is any references.. Great thanks.

**Comment by sushreebarsa** [comment]
@Manal4 Moving this issue to closed status as there has been no recent activity, in case you still face the error please create a new issue.Thanks!
